### ISSUE TITLE
🐛 Multiple pages with Parcel

### ISSUE THREAD

**Issue by zjhch123** [parcel-bundler/parcel#243]
🐛 bug report

I was trying out if it was possible to use multiple pages with Parcel

### Configuration - only these 5 files

```html
<!-- First page -->
<!doctype html>
<html>
<head>
<meta charset="utf-8">
</head>
<body>
  <a href="./index2.html">goto page2</a>
  <script type="text/javascript" src="./index.js"></script>
</body>
</html>
```

```js
// index.js
import util from './util'
util.hi()
```

```html
<!-- Second Page -->
<!doctype html>
<html>
<head>
<meta charset="utf-8">
</head>
<body>
  <a href="./index.html">goto page1</a>
  <script type="text/javascript" src="./index2.js"></script>
</body>
</html>
```

```js
// index2.js
import util from './util.js'
util.hi()
```

```js
// util.js
export default {
  hi() {
    console.log('hi')
  }
}
```

### Command
```
parcel index.html
```

#### First Question
The first page(index.html) can link to second page(index2.html), but in the second page, the tag `<a>` doesn't have the correct attr 'href'.

#### Second Question
In the first page(index.html), I catch this Exception
```
Uncaught Error: Cannot find module '14'
    at newRequire (test-parcel.js:32)
    at localRequire (test-parcel.js:38)
    at Object.require.7../util (test-parcel.js:71)
    at newRequire (test-parcel.js:46)
    at require.7../util (test-parcel.js:63)
    at test-parcel.js:68
```

#### First
the page1 and page2's `<a>` should have correct attr

#### Second
this Exception shouldn't be thrown and 'hi' should be printed

#### First Question
I don't know about this..
#### Second Question
I guess that may be caused by those two files(index.js, index2.js) `import` the same file(util.js)

### 🌍 Your Environment
| Software         | Version(s)
| ---------------- | ----------
| Parcel           |  Latest on master 
| Node             |  v8.9.0
| npm/Yarn         |  yarn 1.2.1
| Operating System |  macOS 10.13

**Comment by DeMoorJasper** [comment]
Tested on Mac OS X High Sierra 10.13.2 and can confirm this doesn't work

**Comment by ssuman** [comment]
I'm working on this.

**Comment by davidnagli** [comment]
Is this a duplicate of #189?

**Comment by DeMoorJasper** [comment]
This is not related to #189 @davidnagli

**Comment by davidnagli** [comment]
Whoops ok I guess I just didn’t understand the issue, they just looked kinda similar.

**Comment by mikekok** [comment]
I'm currently facing the exact same issue as well.

**Comment by devongovett** [comment]
This should be fixed by #380.
